modules/standard/CTypes.chpl

```diff
@@ -1044,6 +1050,13 @@ module CTypes {
     return c_pointer_return_const(x);
   }
 
+  @chpldoc.nodoc
```

Comment:
Shouldn't the version above both of these get deleted?  I'm surprised that compiled okay instead of causing a conflict

Reply:
Which version do you mean?

Reply:
Ohhh, I think I misinterpreted the functions.  There's an `if` statement in both the one on line 1031 and the one on line 1047 about domains, but I think they're really meant to handle other types.

Reply:
Oh, the one on 1031 is `c_addrOf` and the one on 1047 is `c_addrOfConst`

Reply:
Would it make sense to leave a note on the unstable versions to merge them into the other ones when they're stabilized?

Reply:
Good idea will do